### PR TITLE
Prefix tailwind classes

### PR DESCRIPTION
**Pull request by NaveenKharwar**

### CODE REVIEW

**Comment by ajitbohra** [comment]
Looks good we test on production and merge 👍

**Comment by NaveenKharwar** [comment]
> Looks good we test on production and merge 👍

Okay :)

File: includes/views/dashboard/index.php [hunk 1]
```diff
@@ -12,9 +12,9 @@
  */
 
 ?>
-<div class="wrap">
-	<h2 class="flex items-center">
-		<svg class="fill-current w-6 h-6 mr-2" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<div class="tw-wrap">
```
Comment (ajitbohra) [inline includes/views/dashboard/index.php:15]: `<div class="wrap">`

`wrap` is wp-admin class and should not be prefixed!

File: includes/views/dashboard/index.php [hunk 2]
```diff
@@ -24,7 +24,7 @@
 		<?php echo esc_html( get_admin_page_title() ); ?>
 	</h2>
 
-	<div class="flex flex-wrap -mx-2 mt-4">
+	<div class="tw-flex tw-flex-wrap -tw-mx-2 tw-mt-4">
```
Comment (ajitbohra) [inline includes/views/dashboard/index.php:27]: `-tw-mx-2` is not working here it breaks the layout!

these is a visible bug easily noticeable

File: includes/views/import-export/index.php [hunk 3]
```diff
@@ -26,7 +26,7 @@
 		</h2>
 	<?php endif; ?>
 
-	<div class="flex flex-wrap -mx-2 mt-4">
+	<div class="tw-flex tw-flex-wrap -tw-mx-2 tw-mt-4">
```
Comment (ajitbohra) [inline includes/views/import-export/index.php:29]:
```suggestion
	<div class="tw-flex tw-flex-wrap tw--mx-2 tw-mt-4">
```

File: includes/views/settings.php [hunk 4]
```diff
@@ -12,10 +12,10 @@
  */
 
 ?>
-<div class="wrap cmb2-options-page option-<?php echo esc_attr( $cmb_options->option_key ); ?>">
+<div class="tw-wrap cmb2-options-page option-<?php echo esc_attr( $cmb_options->option_key ); ?>">
```
Comment (ajitbohra) [inline includes/views/settings.php:15]:
```suggestion
<div class="wrap cmb2-options-page option-<?php echo esc_attr( $cmb_options->option_key ); ?>">
```

**Pull request merged**